build/scripts/crossgen.sh

```diff
@@ -24,10 +24,10 @@ if [ -z "$RID" ]; then
     fi
 fi
 
-VERSIONS_TARGETS=$CONTAINING_DIR/../Targets/VSL.Versions.targets
-CORECLR_VERSION_SUFFIX="$(grep -o '<CoreClrVersionSuffix>.*</CoreClrVersionSuffix>' $VERSIONS_TARGETS | sed 's/ *<\/*CoreClrVersionSuffix> *//g')"
+DEPENDENCIES=$CONTAINING_DIR/../Targets/Dependencies.props
```

Comment:
Surprised we weren't already seeing errors due to this bad file name given I deleted VSL.Versions.targets a while ago.